cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -49,8 +50,8 @@ public void destroy() {
     // No-op
   }
 
-  protected final WorkflowContext getContext() {
-    return context;
+  public WorkflowContext getContext() {
```

Comment:
I also need it in the executeAction of WorkflowDriver, when I cast the action back in order to get the datasets by using getContext.  That requires the method to be public